ccm_web/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "ccm_web",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "dev": "nodemon --config nmon_config.json server/app.ts",
+    "build-client": "webpack --config webpack/webpack.prod.ts",
+    "build-server": "tsc",
+    "preprod": "npm run build-client && npm run build-server",
+    "prod": "NODE_ENV=production node dist/server/app.js"
```

Comment:
It feels bit weird the `app.js` is in `dist` folder? isn't dist more or less temporary folder?

Reply:
Where would you want to put it? You never edit `app.js` directly, it's always going to be generated by the TypeScript compiler.

Reply:
one a second thought on this, From the `Dockerfile` I can see last line is `CMD node server/app.js` and that is technically starting the server in production mode. is the line `"prod": "NODE_ENV=production node dist/server/app.js"` really called somewhere?

Reply:
This command is to allow to run in production without Docker using `npm run prod`. I'm not using it in Docker because I'm hiding `dist` (coping only the contents of dist into the last stage, not the `dist` directory itself). Does that clarify? I could add a comment somewhere about this if you want.

Reply:
OK, make sense now. Adding a comment would be good.

Reply:
Done, see `README.md`.